test.sh

```diff
@@ -174,12 +174,20 @@ if [[ "$RUN" =~ "vet" ]] ; then
   end_context #vet
 fi
 
+function run_golint() {
+  autogen_regexp="(mockstatsd)"
```

Comment:
I could do it that way for this change! I was thinking we might use this package in other places but that's probably not true.

golint just, by default, doesn't see test files. It wasn't anything we did.